software/digiface.yml

```diff
@@ -0,0 +1,12 @@
+name: Digiface
+website_url: https://ladigitale.dev/digiface/
+description: Create avatars using the Avataaars library (documentation in French).
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - Miscellaneous
+source_code_url: https://codeberg.org/ladigitale/digiface
+demo_url: https://ladigitale.dev/digiface/
+depends_3rdparty: true
```

Comment:
```suggestion
```

I don't think this is necessary, it uses https://github.com/orgordin/vuejs-avataaars which is pulled at build time and is under MIT license, no calls to 3rd-party services are made.